takoyaki/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -17,11 +15,6 @@
  * subsystems, commands, and button mappings) should be declared here.
  */
 public class RobotContainer {
-  // The robot's subsystems and commands are defined here...
-  private final ExampleSubsystem m_exampleSubsystem = new ExampleSubsystem();
-
-  private final ExampleCommand m_autoCommand = new ExampleCommand(m_exampleSubsystem);
-
   /** The container for the robot. Contains subsystems, OI devices, and commands. */
   public RobotContainer() {
```

Comment:
you should
-instantiate an instance of the hopper
-create a command for running the hopper